docs/concepts/01-interfaces.md

```diff
@@ -41,13 +41,13 @@ const link = {
 }
 ```
 
-The `type` and `url` properties there are your own custom API. Slate sees that they exist, but it doesn't ever use them for anything. However, when it goes to render a link element you'll receive an object with the custom properties attached, so that you can render it as:
+The `type` and `url` properties there are your custom API. Slate sees that they exist, but it doesn't ever use them for anything. However, when it goes to render a link element you'll receive an object with the custom properties attached so that you can render it as:
 
 ```jsx
 <a href={element.url}>{element.children}</a>
 ```
 
-When getting started with Slate, it's important to understand all of the interfaces it defines. There are a handful of them that are discussed in each of the guides.
+When getting started with Slate, it's important to understand all of the interfaces it defines. There is a handful of them that are discussed in each of the guides.
```

Comment:
```suggestion
When getting started with Slate, it's important to understand all of the interfaces it defines. There are a handful of interfaces that are discussed in each of the guides.
```

I think are is more accurate... easier to see when changing the word them to interfaces